src/collection/index.js

```diff
@@ -105,6 +105,20 @@ let Collection = function( cy, elements, options ){
   if( createdElements ){
     this.restore();
   }
+
+  // define an iterable behaviour, e.g. for...of, spread syntax
+  this[Symbol.iterator] = function() { // eslint-disable-line no-undef
+    return {
+      next: function() {
+        this._values = this._valueIterator.next();
+        !this._values.done ? this._values.value = this._values.value.ele : null;
+        return this._values;
+      },
+      _valueIterator: map.values(),
+      _values: undefined
+    };
+  };
+
```

Comment:
It would be better to use an index rather than depending on `values()` working for another object.  Here's an example:

https://github.com/cytoscape/cytoscape.js/blob/v3.12.1/src/util/hash.js#L22-L40